ops/model.py

```diff
@@ -531,10 +529,10 @@ def __repr__(self):
     def is_leader(self) -> bool:
         """Return whether this unit is the leader of its application.
 
-        This can only be called for your own unit.
+        This can only be called for the charm's own unit.
```

Comment:
Similar here and below: the other `Unit` instance may be a `Unit` instance from this charm. I think probably "current unit" throughout, unless you can think of a better phrase.